### PR TITLE
Add startup documentation

### LINKED ISSUE #88: Improve startup documentation
Explain better how to start the API. See #85

### PR DESCRIPTION
**Pull request by gcotelli**
Fixes #88

### CODE REVIEW

File: docs/Startup.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Starting up the API
```
Comment (fortizpenaloza) [inline docs/Startup.md]: Starting up an API?

File: docs/Startup.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    PetOrdersRESTfulController new
+    }.
+
+api
+    install;
+    start
+```
+will install the example controllers, and serve the API in the local machine using the port 9999.
+
+The configuration parameters are passed to `Teapot` so you can configure here any of the options accepted by `Teapot` or `Zinc` servers. The `operations` key is mandatory and it's used for the plugin system of Stargate. See [the operations and plugins documentation](Operations.md) to get a list of valid options. For deployment situations is recommended to use the `jwt` authentication scheme.
```
Comment (fortizpenaloza) [inline docs/Startup.md]:
```suggestion
The configuration parameters are passed to `Teapot` so you can configure here any of the options accepted by `Teapot` or `Zinc` servers. The `operations` key is mandatory and it's used for the plugin system of Stargate. See [the operations and plugins documentation](Operations.md) to get a list of valid options. For deployment environments, the `jwt` authentication scheme is recommended.
```

File: docs/Startup.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+`#serverUrl` parameter is used in Stargate as the base URL in the media controls. So, if you're deploying your API behind a proxy and using an specific domain, this must be reflected in the configuration so the media control links are generated properly. For example `#serverUrl -> 'http://api.example.com'`.
```
Comment (fortizpenaloza) [inline docs/Startup.md]:
```suggestion
`#serverUrl` parameter is used as the base URL in the media controls. So, if you're deploying your API behind a proxy and using a specific domain, this must be reflected in the configuration so the media controls links are generated properly. For example `#serverUrl -> 'http://api.example.com'`.
```

**Review by fortizpenaloza — CHANGES_REQUESTED** [review]
Good job @gcotelli !

File: docs/Startup.md (hunk 1 above)
Comment (gcotelli) [inline docs/Startup.md]:
```suggestion
# API start up
```

**Pull request merged**